src/main/java/edu/ie3/util/geo/GrahamScan.java

```diff
@@ -1,240 +0,0 @@
-/*
- * © 2020. TU Dortmund University,
- * Institute of Energy Systems, Energy Efficiency and Energy Economics,
- * Research group Distribution grid planning and operation
-*/
-package edu.ie3.util.geo;
-
-import java.awt.*;
-import java.util.*;
-import java.util.List;
-
-/** */
-public final class GrahamScan {
```

Comment:
Why do you want to remove these methods? They aren't event marked as deprecated ....

Reply:
Bc they have solely been used in GeoUtils. I think I will then restore the GeoUtils and try to improve the visibility of the cause of deprecation.